### PR TITLE
mount: add --mkdir example

### PR DESCRIPTION
**Pull request by CyrusYip**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/mount.md [hunk 1]
```diff
@@ -11,6 +11,10 @@
 
 `mount -t {{filesystem_type}} {{path/to/device_file}} {{path/to/target_directory}}`
 
+- Create the target directory if it does not exist and mount a device to it:
```
Comment (EmilySeville7cfg) [inline pages/common/mount.md:14]:
```suggestion
- Create a specific directory if it does not exist and mount a device to it:
```

Optional change: `a specific` may sound better.

**Review by EmilySeville7cfg — APPROVED** [review]
- No mandatory changes found, approved.
- Thanks for a contribution! 🚀

**Comment by blueskyson** [comment]
@CyrusYip Have you signed the CLA license?

**Comment by CyrusYip** [comment]
@blueskyson I signed in my last pull request (https://github.com/tldr-pages/tldr/pull/9521). Do I need to sign in every pull request?

**Comment by kbdharun** [comment]
>  I signed in my last pull request (#9521). Do I need to sign in for every pull request?

Signing the CLA once is enough. Sorry for the inconvenience this is a known Bug with CLA assistant.

> @CyrusYip Have you signed the CLA license?

@blueskyson this bug occasionally occurs when some unknown event in GitHub API causes this error (Expected — Waiting for status to be reported), for PRs of this kind we generally force merge it. Organizational owners can do force merging I think so cc @navarroaxel and @CleanMachine1.

**Comment by blueskyson** [comment]
Reopening the PR fixed it.

**Comment by kbdharun** [comment]
> Reopening the PR fixed it.

Nice

**Pull request merged**